### DISCUSSION TITLE
Found the reason why ruins lag so much

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#15118] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

It's alien turret targeting items 

![20241112073515_1](https://github.com/user-attachments/assets/c8e94cae-4b11-4b23-897d-a4210a09ca87)
![20241112073539_1](https://github.com/user-attachments/assets/547b57dc-2223-40fa-a7b2-4a36b8c03329)
![20241112073551_1](https://github.com/user-attachments/assets/5a2ac579-4731-40a8-8267-1ec6691f050e)
![20241112073655_1](https://github.com/user-attachments/assets/e0e0146e-e015-45a2-8dec-f952c0b9cf59)
![20241112074044_1](https://github.com/user-attachments/assets/3d00f0a6-f4e2-4454-b27f-f40d48f788ca)

### Reproduction steps

quickstart dugong 100
until ruins with turrets get spawned

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in the latest unstable build (PR in our private repo that optimized this https://github.com/FakeFishGames/Barotrauma-development/pull/5909).

Btw, what's that debug view you're using here?

**Reply by SomeRandomNoobKekeke** [reply]
https://steamcommunity.com/sharedfiles/filedetails/?id=3309924659
